app/controllers/solidus_square/webhooks_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module SolidusSquare
+  class WebhooksController < BaseController
+    def update
+      ::SolidusSquare::Webhooks::Sorter.call(params)
+
+      render json: {}, status: :ok
+    rescue ActiveRecord::RecordInvalid => e
```

Comment:
Why you are only rescuing the ActiveRecord::RecordInvalid? 
I think you should rescue all the StandardError here.